packages/lesswrong/components/collections/CollectionsTooltip.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { ReactNode } from "react";
```

Comment:
This is the renamed `CollectionsHoverOver` component, but there are enough changes that git thinks it's a completely different file. The only place this is used is for the EA handbook on the best of page (LW don't use it at all).